src/resolvers-cassandra/Settings/index.js

```diff
@@ -4,7 +4,8 @@ const mutations = require('./mutations');
 const queries = require('./queries');
 
 module.exports = {
-  createOrReplaceSite: mutations.createOrReplaceSite,
+  createSite: mutations.createSite,
+  removeSite: mutations.removeSite,
```

Comment:
Done in [b34b9cf](https://github.com/CatalystCode/project-fortis-services/pull/77/commits/b34b9cf10993e2f8ae6c1eecc15ddf20d3aecab2)